lib/ansible/plugins/action/__init__.py

```diff
@@ -29,7 +31,7 @@
 from ansible.utils.display import Display
 from ansible.utils.unsafe_proxy import wrap_var
 from ansible.vars.clean import remove_internal_keys
-
+from distutils.version import LooseVersion
```

Comment:
`LooseVersion` or anything from `distutils.version` is problematic on Python3. I haven't fully understood how this is being used yet, but due to more strict type comparisons in Python3, comparing `LooseVersion` can result in a traceback if you compare a string and an integer.  See https://github.com/ansible/ansible/pull/34427

I had an implementation of `SafeLooseVersion` that would cast all values to strings after `LooseVersion` did its parsing.

Something like:

```
    def parse(self, vstring):
        super(SafeLooseVersion, self).parse(vstring)
        self.version[:] = [str(v) for v in self.version]
```